### PR TITLE
Token verification cache should never fail

### PR DESCRIPTION
**Pull request by leplatrem**
Mozilla/PyFxA#48

### CODE REVIEW

File: kinto_fxa/authentication.py [hunk 1]
```diff
@@ -25,13 +25,23 @@ def __init__(self, cache, ttl):
         self.ttl = ttl
 
     def get(self, key):
-        return self.cache.get(key)
+        try:
+            return self.cache.get(key)
+        except:
+            logger.exception("Error while fetching from cache")
```
Comment (Natim) [inline kinto_fxa/authentication.py]: Please also concatenate the exception error message for debugging.

File: kinto_fxa/authentication.py [hunk 2]
```diff
@@ -25,13 +25,23 @@ def __init__(self, cache, ttl):
...
+        return None
 
     def set(self, key, value):
-        self.cache.set(key, value, self.ttl)
+        try:
+            self.cache.set(key, value, self.ttl)
+        except:
+            logger.exception("Error while storing in cache")
```
Comment (Natim) [inline kinto_fxa/authentication.py]: ditto

File: kinto_fxa/authentication.py [hunk 3]
```diff
@@ -25,13 +25,23 @@ def __init__(self, cache, ttl):
...
 
     def delete(self, key):
-        self.cache.delete(key)
+        try:
+            self.cache.delete(key)
+        except:
+            logger.exception("Error while deleting from cache")
```
Comment (Natim) [inline kinto_fxa/authentication.py]: ditto

**Comment by leplatrem** [comment]
I don't think the concatenation is necessary:

```
>>> import logging
>>> logging.basicConfig()
>>> logger = logging.getLogger()
>>> try:
...     1/0
... except:
...     logger.exception("hoho")
... 
ERROR:root:hoho
Traceback (most recent call last):
  File "<stdin>", line 2, in <module>
ZeroDivisionError: integer division or modulo by zero
```

**Pull request merged**